distribution/linux/openrct2.appdata.xml

```diff
@@ -63,4 +63,8 @@
     <content_attribute id="money-purchasing">none</content_attribute>
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
+  <releases>
+      <release version="0.2.6" date="2020-04-17" />
+      <release version="0.2.5" date="2020-03-24" />
```

Comment:
I don't know how important a complete release history is, but the dates are listed here: https://github.com/OpenRCT2/OpenRCT2/releases

Reply:
I think it's used to describe updates from one version with the information to the next, so I don't think it's super important to have the whole back catalogue. But my knowledge of this is rather vague - if one of the Flathub maintainers tells me it needs the full list, I'd accept that (with some grumbling).